app/models/immunisation_import.rb

```diff
@@ -68,6 +68,13 @@ def process!(patient_session:)
     parse_rows! if rows.nil?
     return if invalid?
 
+    rows
+      .map(&:to_location)
+      .uniq(&:urn)
+      .reject(&:invalid?)
+      .map(&:attributes)
+      .tap { Location.create! _1 }
```

Comment:
Just realised we should set `imported_from` and add the field to locations.

Reply:
Are we waiting for this to be done?

Reply:
Sorted now